src/gapstate.h

```diff
@@ -35,11 +35,6 @@ typedef struct GAPState {
 #endif
 
     /* From intrprtr.c */
-    UInt IntrIgnoring;
-    UInt IntrReturning;
-    UInt IntrCoding;
-    Obj  IntrState;
-    Obj  StackObj;
```

Comment:
These variables are removed from the global GAPState and put into a stack allocated `IntrState`.

WEll, all except `STATE(IntrState)` which itself became superfluous.